src/Engine/Manager/ResourceManager.hpp

```diff
@@ -23,138 +23,138 @@ class ScriptFile;
 /// Handles all resources.
 class ResourceManager {
     friend class Hub;
-    
+
     public:
-        /// Constructor
-        ResourceManager() {}
+    /// Constructor
+    ResourceManager() {}
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Member variable 'ResourceManager::rectangle' is not initialized in the constructor. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=cppcheck%3AuninitMemberVar)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Member variable 'ResourceManager::cube' is not initialized in the constructor. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=cppcheck%3AuninitMemberVar)